plugins/test-runners/pytest/cosmic_ray_pytest_runner/runner.py

```diff
@@ -29,16 +27,32 @@ class PytestRunner(TestRunner):
     """
 
     def _run(self):
+        from io import StringIO
```

Comment:
Can this import be at the top of the module? That would be more idiomatic, less surprising, etc.

Reply:
Sure, will do it for now (but only later) - in the long run this should go/come with `TestRunner` likely anyway.